src/components/cylc/tree/index.js

```diff
@@ -59,12 +70,17 @@ function convertGraphQLWorkflowToTree (workflow) {
     } else {
       if (!lookup.get(parent.id)) {
         // construct preliminary parent node if not root
-        Object.assign(parent, {
+        const node = Object.assign({}, parent, {
           __type: 'family',
           name: parent.name,
-          children: []
+          children: [],
+          size: 32,
```

Comment:
What does the `size` bit do? Should consider making this a constant for easy fiddling.

Is that a visual dimension or to do with the number of nodes in infinite tree?

Reply:
Good spot. Deservers some documentation and we can move to a constant I think. That's a required property for the `InfiniteTree` strategy used.

This tells the library what is the size of each element added to the infinite tree. And is later used by the library to calculate how many elements will fit into the screen and calculate the buffer size too.

Reply:
Aah so it's a height estimate in pixels?

Reply:
Exactly 👍